.changeset/nasty-points-wait.md

```diff
@@ -0,0 +1,5 @@
+---
+"@hashicorp/design-system-components": minor
+---
+
+`Dropdown` - Converted component to TypeScript
```

Comment:
Did a quick search, and seems the `MenuPrimitive` is publicly used: https://github.com/search?q=MenuPrimitive+user%3Ahashicorp+repo%3Ahashicorp%2Fcloud-ui&type=code&ref=advsearch

```suggestion
`Dropdown` - Converted component to TypeScript

`MenuPrimitive` - Converted component to TypeScript
```

Reply:
they don't seem to be using it anymore, so maybe it is a good time to start pulling the plug on `MenuPrimitive` (if we decide so)